model/vm.rb

```diff
@@ -67,17 +67,6 @@ def display_state
     super
   end
 
-  def mem_gib_ratio
```

Comment:
I think this is still used in later commits.

Reply:
Although, I was thinking of moving it to VmHost class, as it is no longer needed in Vm. It is used when allocating a VmHostSlice, as we cannot use @request.memory because that is a size of memory for a Vm not for the slice.

Reply:
I saw your 2nd comment after I applied your 1st comment. For the 2nd comment, let's move it to `VmHost` in the other PR which allocates a VmHostSlice so we see how it's used.